sdks/python/apache_beam/internal/metrics/metric.py

```diff
@@ -262,4 +262,8 @@ def bigtable_error_code_to_grpc_status_string(grpc_status_code):
     if (grpc_status_code is not None and
         grpc_status_code in grpc_to_canonical_gcp_status):
       return grpc_to_canonical_gcp_status[grpc_status_code]
+    if grpc_status_code is None:
+      # Bigtable indicates this can be retried but itself has exhausted retry
+      # timeout or there is no retry policy set for bigtable.
+      return grpc_to_canonical_gcp_status[4]
```

Comment:
I do not fully understand why are we mapping this to deadline exceeded?

Reply:
The behavior of the bigtable client is retrying until it exhausts all the retry budget. There isn't a canonical status to notify the client to retry. "UNAVAILABLE" indicates the request is retryable but shouldn't apply here. So I think "DEADLINE_EXCEEDED" is a better fit here.